packages/osd-dev-utils/certs/README.md

```diff
@@ -28,7 +28,7 @@ The password used for both of these is "storepass". Other copies are also provid
 
 ## Certificate generation
 
-[OpenSearch cert-util](https://www.opensearch.org/guide/en/elasticsearch/reference/current/certutil.html) and [OpenSSL](https://www.openssl.org/) were used to generate these certificates. The following commands were used from the root directory of OpenSearch:
+[OpenSearch cert-util](https://opensearch.org/docs/latest/security-plugin/configuration/generate-certificates/) and [OpenSSL](https://www.openssl.org/) were used to generate these certificates. The following commands were used from the root directory of OpenSearch:
```

Comment:
Note: we don't have a cert-util. We should unlink that part but add a link to the self-service cert generation.